chat/command.go

```diff
@@ -190,8 +191,16 @@ func InitCommands(c *Commands) {
 				if cfg.Theme != nil {
 					theme = cfg.Theme.ID()
 				}
-				body := fmt.Sprintf("Current theme: %s", theme)
-				room.Send(message.NewSystemMsg(body, user))
+				var output bytes.Buffer
+				fmt.Fprintf(&output, "Current theme: %s%s", theme, message.Newline)
```

Comment:
I mean it's kinda the equivalent of writing ("Current theme: %s%s", foo, "\n") which feels weird to me vs ("Current theme: %s\n", foo).

I'll merge it either way, just let me know when it's ready for merging. :)